tests/test_jitterbuffer.py

```diff
@@ -176,6 +203,26 @@ def test_remove(self):
         self.assertEqual(jbuffer._origin, 3)
         self.assertPackets(jbuffer, [None, None, None, 3])
 
+    def test_smart_remove(self):
+        jbuffer = JitterBuffer(capacity=4)
+
+        jbuffer.add(RtpPacket(sequence_number=0, timestamp=1234))
+        # jbuffer.add(RtpPacket(sequence_number=1, timestamp=1234))
```

Comment:
Please remove the commented line

Reply:
No problem I'll remove it - I just thought it's easier to understand this way.